File: src/WebUI/src/models/character-statistics.ts
import CharacterAttributes from '@/models/character-attributes';
import CharacterSkills from '@/models/character-skills';
import CharacterWeaponProficiencies from '@/models/character-weapon-proficiencies';

export default class CharacterStatistics {
  public attributes: CharacterAttributes;
  public skills: CharacterSkills;
  public weaponProficiencies: CharacterWeaponProficiencies;
}

File: src/Domain/Entities/CharacterStatistics.cs
namespace Crpg.Domain.Entities
{
    public class CharacterStatistics
    {
        public CharacterAttributes Attributes { get; set; } = new CharacterAttributes();
        public CharacterSkills Skills { get; set; } = new CharacterSkills();
        public CharacterWeaponProficiencies WeaponProficiencies { get; set; } = new CharacterWeaponProficiencies();
    }
}